Formula/clickhouse-odbc.rb

```diff
@@ -0,0 +1,104 @@
+class ClickhouseOdbc < Formula
+  desc "Official ODBC driver implementation for accessing ClickHouse as a data source"
+  homepage "https://github.com/ClickHouse/clickhouse-odbc#readme"
+  url "https://github.com/ClickHouse/clickhouse-odbc.git",
+      tag:      "v1.1.10.20210822",
+      revision: "c7aaff6860e448acee523f5f7d3ee97862fd07d2"
+  license "Apache-2.0"
+  head "https://github.com/ClickHouse/clickhouse-odbc.git", branch: "master"
+
```

Comment:
```suggestion

  livecheck do
    url :stable
    regex(/^v?(\d+(?:\.\d+)+)$/i)
  end

```

This repository also contains older date-based tags that are treated as newer from the standpoint of `Version` comparison, so `brew livecheck` will see `2018-09-03` as newest instead of `1.1.10.20210822` (2018 > 1). Using the standard regex for Git tags like `1.2.3`/`v1.2.3` (in the suggestion above) will restrict matching to the correct versions in this context.

Reply:
Thanks.